### ISSUE TITLE
Titrari provider HTTPError

### ISSUE THREAD

**Issue by ciotorcristian** [morpheus65535/bazarr#2709]
**Describe the bug**

Titrari provider is failing with the following error message:

`Using titrari again after 10 minutes, (disabled because: HTTPError) | 2024-10-09 10:32:35 |  
  Throttling titrari for 10 minutes, until 24/10/09 09:38, because of: HTTPError. Exception info: "'404 Client Error: Not Found for url: https://www.titrari.ro/?page=maicauta&z7=&z2' ~ titrari.py@150" | 2024-10-09 09:28:41 |  
  Using titrari again after 10 minutes, (disabled because: HTTPError) | 2024-10-09 09:20:42 |  
  Throttling titrari for 10 minutes, until 24/10/09 03:39, because of: HTTPError. Exception info: "'404 Client Error: Not Found for url: https://www.titrari.ro/?page=maicauta&z7=&z2' ~ titrari.py@150" | 2024-10-09 03:29:14`

If further info are needed please tell me and i will try to gather it.

**To Reproduce**
1. Try to download any subtitle in romanian
2. Receive the above errors

**Expected behavior**
Titrari provider to not throw errors.

**Software (please complete the following information):**
Bazarr Version 1.4.5
Package Version v1.4.5-ls271 by linuxserver.io
Sonarr Version 4.0.9.2244
Radarr Version 5.11.0.9244
Operating System Linux-6.1.0-25-amd64-x86_64-with

**Comment by morpheus65535** [comment]
The provided url is working as expected. It's either a temporary issue or something on your side. About 2k subtitles get downloaded from this provider daily.

**Comment by ciotorcristian** [comment]
I tried to narrow the problem down. I have started the upgrade already downloaded subtitles task and after ~100 requests, the errors appear again. When i go to the URL in the error i get the following message: Too many requests. Fuck off!

It seems that the site is refusing any more requests because too many were sent in a short period of time. Is there anything that bazzar can to to prevent this?
